pom.xml

```diff
@@ -297,7 +297,7 @@
 
         <formatterConfigPath>formatter.xml</formatterConfigPath>
         <formatterGoal>validate</formatterGoal>
-        <oldMuleArtifactVersion>4.2.0-SNAPSHOT</oldMuleArtifactVersion>
+        <oldMuleArtifactVersion>4.2.1-SNAPSHOT</oldMuleArtifactVersion>
```

Comment:
Is this still necessary? This was probably already fixed.

Reply:
Indeed.